### PR TITLE
Annotate nodes

### PR DESCRIPTION
**Pull request by finnagin**
Made this and annotate_edges as a suggestion for being able to just annotate edges or nodes.

### CODE REVIEW

File: docs/annotate_nodes.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+### allowed changes
+
+- add attributes to knodes
+
+### parameters
+
+```yaml
+properties:
+  attributes:
+    description: A list of attributes to anotate the nodes with. If not included then
```
Comment (dkoslicki) [inline docs/annotate_nodes.md:28]: Same comment as in #42 re: enumerating possibilities here

**Review by dkoslicki — CHANGES_REQUESTED** [review]

File: operations/annotate_nodes.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  output: annotation/messages/03_annotated_nodes.json
+input_requirements: []
+output_guarantees: []
+allowed_changes:
+- add attributes to knodes
+parameters:
+  type: object
+  properties:
+    attributes:
+      description: A list of attributes to anotate the nodes with. If not included then all available data will be anotated.
```
Comment (dkoslicki) [inline operations/annotate_nodes.yml:17]: ...description: A list of attribute type ids...

File: docs/annotate_nodes.md (hunk 1 above)
Comment (dkoslicki) [reply docs/annotate_nodes.md:28]: Decision: no, the UI can pull this from the endpoints themselves

File: operations/annotate_nodes.yml (hunk 2 above)
Comment (patrickkwang) [reply operations/annotate_nodes.yml:17]:
```suggestion
      description: A list of attributes to annotate the nodes with. If not included then all available data will be annotated.
```

**Review by patrickkwang — CHANGES_REQUESTED** [review]

File: operations/annotate_nodes.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+input_requirements: []
+output_guarantees: []
+allowed_changes:
+- add attributes to knodes
+parameters:
+  type: object
+  properties:
+    attributes:
+      description: A list of attributes to annotate the nodes with. If not included then all available data will be annotated.
+      type: array
```
Comment (patrickkwang) [inline operations/annotate_nodes.yml:18]:
```suggestion
      type: array
      items:
        type: string
```

**Review by patrickkwang — CHANGES_REQUESTED** [review]

**Review by patrickkwang — APPROVED** [review]

**Review by dkoslicki — APPROVED** [review]

**Pull request merged**